.github/workflows/tox_tests.yml

```diff
@@ -0,0 +1,87 @@
+name: Tox tests on SDK packages
```

Comment:
Overall general question:
- Why is there a split of jobs like linux, mac, windows each testing only specific versions of Python. eg py27 only on linux, windows tests only for py37, py38? Can we try to keep it under a single block of def while specifying OS platforms and py env (https://github.com/alexa/ask-sdk-controls/blob/develop/.github/workflows/unit-test.yml#L13)

Reply:
Yeah, that was my plan as well. To include the whole matrix and then use `exclude` to exclude specific python versions. However, that lead to issues with `tox-env` variable being passed to each `os`. Since adding a list of configuration variables in a matrix will lead to all combinations (3 operating systems, 4 python versions, 8 tox envs), it resulted in more `excludes` and conditional checks (eg: tox-env = py36-mywin, os = ubuntu-latest etc.). After a bunch of tries, I came back to this approach which is simpler to handle and maintain.

Reply:
Ah i see, why do we need to pass `tox-env` OS specific since we are using Github Actions, can't execute the mypy and test commands defined here? Again I'm ok with this change too.
Was just curious 👍

Reply:
Yeah, good point. I would love to leave `tox` available for local testing until we get some support for running `github actions` locally. It's probably easier for users to run tests on the repo directly through the tox command.